tech.json

```
@@ -34,6 +34,12 @@
             }
           ]
         },
+        {
+          "name": "canRelyOnLuck",
+          "techRequires": [],
+          "otherRequires": [],
+          "note": "Executing a strat that has a high chance to fail, even with good execution, due to random elements such as enemy drops, and failure requires at minimum resetting the room."
```

Maybe something like this?
```suggestion
          "note": [
            "Executing a strat that has a reasonable chance to fail due to random elements such as enemy drops.",
            "This is used where failure is unforgiving and may result in a death or a slow process to reattempt."
          ]
```